operator/v1/types_console.go

```diff
@@ -143,8 +143,42 @@ type Capability struct {
 	Visibility CapabilityVisibility `json:"visibility"`
 }
 
+// ThemeType defines the type of the theme for the console UI.
+type ThemeType string
+
+// ThemeType values
+const (
+	// DarkTheme represents the dark theme for the console UI.
+	DarkTheme ThemeType = "Dark"
+	// LightTheme represents the light theme for the console UI.
+	LightTheme ThemeType = "Light"
+)
+
+// CustomFiles defines a configuration based on theme types for the console UI logo.
+type CustomFiles struct {
+	// type of the theme for the console UI.
+	// +unionDiscriminator
+	// +kubebuilder:validation:Enum:="Dark";"Light"
```

Comment:
This is currently limited to either `Dark` or `Light`. What if a user wants to use the same logo on both dark and light themes?